### PR TITLE
EM: introduce crossCloudAzureActiveDirectoryTenant in beta

### PR DESCRIPTION
**Pull request by markwahl-msft**
This change introduces a new subtype of identitysource, crossCloudAzureActiveDirectoryTenant. This type is similar to  azureActiveDirectoryTenant but can be used to refer to an azure AD tenant in another cloud (e.g., gov, china, commercial). There are no new APIs or addressable resources with this change.

### CODE REVIEW

File: api-reference/beta/resources/crosscloudazureactivedirectorytenant.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+Used in the identity sources of an [connectedOrganization](connectedOrganization.md). The `@odata.type` value `#microsoft.graph.crossCloudAzureActiveDirectoryTenant` indicates that this type identifies another Azure Active Directory tenant, in a different cloud, as an identity source for a connected organization.
+
+
+## Properties
+
+| Property                     | Type                      | Description |
+| :--------------------------- | :------------------------ | :---------- |
+| displayName |String | The name of the Azure Active Directory tenant. Read only. |
+| tenantId |String | The ID of the Azure Active Directory tenant. Read only. |
+| cloudInstance | String | The ID of the cloud where the tenant is located, one of `microsoftonline.com`, `microsoftonline.us` or `partner.microsoftonline.cn`. Read only. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/crosscloudazureactivedirectorytenant.md:25]: Alphabetical order, please.

File: api-reference/beta/resources/crosscloudazureactivedirectorytenant.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+doc_type: "resourcePageType"
+---
+
+# crossCloudAzureActiveDirectoryTenant resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Used in the identity sources of an [connectedOrganization](connectedOrganization.md). The `@odata.type` value `#microsoft.graph.crossCloudAzureActiveDirectoryTenant` indicates that this type identifies another Azure Active Directory tenant, in a different cloud, as an identity source for a connected organization.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/crosscloudazureactivedirectorytenant.md:16]: Minor fix
```suggestion
Used in the identity sources of a [connectedOrganization](connectedOrganization.md) object. The `@odata.type` value `#microsoft.graph.crossCloudAzureActiveDirectoryTenant` indicates that this type identifies another Azure AD tenant in a different cloud as an identity source for a connected organization.
```

**Review by FaithOmbongi — APPROVED** [review]
Thank you @markwahl-msft for your work in this PR. Approved with suggestions for minor fixes.

File: api-reference/beta/resources/crosscloudazureactivedirectorytenant.md (hunk 1 above)
Comment (markwahl-msft) [reply api-reference/beta/resources/crosscloudazureactivedirectorytenant.md]: updated this list, thanks.

**Pull request merged**